libs/external-db-logger/src/logger.ts

```diff
@@ -0,0 +1,36 @@
+import pino from 'pino'
+import { ILogger, LogLevel, Data, LoggerError } from './types'
+
+export class Logger implements ILogger {
+    private logLevel: LogLevel
+    private logger: pino.Logger
+
+  constructor(level = LogLevel.DEBUG) {
+    this.logLevel = level
+    this.logger = pino({
+        base: {},
+        timestamp: false,
+        level,
+        messageKey: 'message',
+        formatters: {
+          level: (label) => ({ level: label.toUpperCase() }),
+        },
+      })
+  }
+
+  debug(message: string, data?: Data): void {
+    this.logger.debug({ data }, message)
+  }
+
+  info(message: string, data?: Data): void {
+    this.logger.info({ data }, message)
```

Comment:
any other global meta-data we want to add?
maybe the DC? some cloud-run meta-data the could be of interest?

Reply:
Oops, I missed it
At the moment I can't think of such useful information